storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3819,7 +3820,11 @@ void ha_rocksdb::update_auto_incr_val() {
 
   my_bitmap_map *const old_map =
       dbug_tmp_use_all_columns(table, table->read_set);
-  new_val = field->val_int() + 1;
+  new_val = field->val_int();
+  if (new_val + 1 > new_val) {  // don't increment if we would wrap around
```

Comment:
...whereas the next snippet checks against "std::numeric_limits<longlong>::max()" without UB